src/Sulu/Bundle/AdminBundle/Resources/js/components/Popover/Popover.js

```diff
@@ -107,20 +107,9 @@ class Popover extends React.Component<Props> {
         const {
             offsetWidth,
             offsetHeight,
-            scrollWidth,
-            scrollHeight,
-            clientWidth,
-            clientHeight,
         } = this.popoverChildRef;
 
-        // calculating real size by considering borders, margins and paddings
-        const outerWidth = scrollWidth + offsetWidth - clientWidth;
-        const outerHeight = scrollHeight + offsetHeight - clientHeight;
-
-        this.setPopoverSize(
-            outerWidth,
-            outerHeight
-        );
+        this.setPopoverSize(offsetWidth, offsetHeight);
```

Comment:
If I don't make this change, then the dropdown shows on top if the collaboration toolbar is shown. But when I make this change, some edge cases in the `SingleSelect` component do not work correctly anymore... I would seriously suggest to remove that functionatlity that centers the menu over the select...